pkg/term/term_windows.go

```diff
@@ -1,12 +1,16 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	"github.com/docker/docker/pkg/term/windows"
+	"github.com/docker/docker/vendor/src/github.com/Azure/go-ansiterm/winterm"
```

Comment:
Yes, the `GOPATH` on the build env is set to `/go:/go/src/github.com/docker/docker/vendor`